actiontext/lib/action_text/attribute.rb

```diff
@@ -41,13 +41,16 @@ module Attribute
       #     `strict_loading:` will be set to the value of the
       #     `strict_loading_by_default` class attribute (false by default).
       #
+      # *   `:store_if_blank` - Pass false to not create RichText records with empty values,
+      #     if a blank value is provided. Default: true.
+      #
       #
       # Note: Action Text relies on polymorphic associations, which in turn store
       # class names in the database. When renaming classes that use `has_rich_text`,
       # make sure to also update the class names in the
       # `action_text_rich_texts.record_type` polymorphic type column of the
       # corresponding rows.
-      def has_rich_text(name, encrypted: false, strict_loading: strict_loading_by_default)
+      def has_rich_text(name, encrypted: false, strict_loading: strict_loading_by_default, store_if_blank: true)
```

Comment:
_Maybe_ `allow_blank` to mirror validations?
```suggestion
      def has_rich_text(name, encrypted: false, strict_loading: strict_loading_by_default, allow_blank: true)
```

If people think that's too confusing, maybe

```suggestion
      def has_rich_text(name, encrypted: false, strict_loading: strict_loading_by_default, store_blank: true)
```

Reply:
I considered that, but to me that implies that blank values would cause a validation error, which isn’t the case.